CHANGELOG.md

```diff
@@ -22,11 +22,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Breaking changes
```

Comment:
👍

Reply:
File a to do / add comments for when deprecation will be removed? (See styleOptions deprecation warnings)